index.html

```diff
@@ -93,5 +95,11 @@
            some of the bootstrap css -->
       <link rel="stylesheet" type="text/css" media="all"
             href="style.css" />
+            <div class="figure">
+    <figure>
```

Comment:
Are you satisfied with the way this image looks? try looking at it in a full-width window.